src/gui/folderwizard/folderwizard.h

```diff
@@ -15,17 +15,19 @@
 #ifndef MIRALL_FOLDERWIZARD_H
 #define MIRALL_FOLDERWIZARD_H
 
-#include <QWizard>
 #include <QNetworkReply>
 #include <QTimer>
+#include <QWizard>
 
-#include "folder.h"
 #include "accountfwd.h"
 
-#include "ui_folderwizardsourcepage.h"
-#include "ui_folderwizardtargetpage.h"
+#include "gui/folder.h"
 
 class QCheckBox;
+class QTreeWidgetItem;
```

Comment:
Do we need these forward declarations? I guess we could just `#include` these classes.